lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -4,13 +4,10 @@ hasClass = (node, search) ->
 nodeIsDisabled = (node) ->
    node.getAttribute('disabled') || hasClass(node, 'disabled')
 
-TurboGraft.handlers.remoteMethodHandler = (ev) ->
-  target = ev.clickTarget
-  httpRequestType = target.getAttribute('tg-remote')
-  return unless httpRequestType
-  ev.preventDefault()
-  httpUrl = target.getAttribute('href')
-  throw new Error("Turbograft developer error: You did not provide a URL ('href' attribute) for tg-remote") unless httpUrl
+setupRemoteFromTarget = (target, httpRequestType, urlAttribute, form = null) ->
+  httpUrl = target.getAttribute(urlAttribute)
+
+  throw new Error("Turbograft developer error: You did not provide a URL ('#{urlAttribute}' attribute) for tg-remote") unless httpUrl
 
   if target.getAttribute("remote-once")
     target.removeAttribute("remote-once")
```

Comment:
can you change this line to `target.removeAttribute("remote-once") if target.getAttribute("remote-once")` while you're in here?  I think that's a bit nicer